tests/core/daemon/test_daemon.py

```diff
@@ -102,6 +139,57 @@ def assert_response(response: aiohttp.http_websocket.WSMessage, expected_respons
     assert message["data"] == expected_response_data
 
 
+def assert_running_services_response(response_dict: Dict[str, Any], expected_response_dict: Dict[str, Any]) -> None:
+    for k, v in expected_response_dict.items():
+        if k == "running_services":
+            # Order of services is not guaranteed
+            assert len(response_dict[k]) == len(v)
+            assert set(response_dict[k]) == set(v)
+        else:
+            assert response_dict[k] == v
+
+
+def assert_is_running_response(response_dict: Dict[str, Any], expected_response_dict: Dict[str, Any]) -> None:
+    for k, v in expected_response_dict.items():
+        assert response_dict[k] == v
+
+
+@pytest.fixture(scope="session")
+def mock_lonely_daemon():
```

Comment:
I personally wouldn't call this a mock since we aren't messing with any existing code, just passing a different thing to it.  But, I haven't reviewed the terminology around this recently.

Reply:
Punting per an offline discussion.